src/controllers/authentication.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-unsafe-optional-chaining */
+import express from 'express'
+import User from '../database/models/userModel'
+import Token from '../database/models/tokenModel'
+
+const verifyEmail = async (req: express.Request, res: express.Response) => {
+  try {
+    const { token } = req.params
+
+    if (!token) {
+      return res.status(400).send({ message: 'Bad request' })
+    }
+    const myToken = await Token.findOne({ where: { token } })
```

Comment:
So the reason we use a JWT is also to be able to verify the authenticity of the request, meaning we need to verify that the token is from the right use. jwt.verify()